### ISSUE TITLE
Pull request comment appearing even though it's disabled in codecov.yaml

### ISSUE THREAD

**Issue by sampsyo** [codecov/support#162]
We're getting pull request comments even though we've disabled them in our YAML configuration.

Here's an example: https://github.com/beetbox/beets/pull/1961#issuecomment-214909579

Here's our configuration: https://github.com/beetbox/beets/blob/master/codecov.yaml

What else do we need to do?

**Comment by untitaker** [comment]
They also magically appeared on https://github.com/pimutils/vdirsyncer, even though they were disabled in the admin interface.

**Comment by sampsyo** [comment]
@untitaker, I think the problem is that Codecov just did a complete overhaul that did away with the admin interface settings. This means all our settings were silently reset to the defaults and it's up to us to figure out the new in-repo configuration scheme in a hurry. 😒

**Comment by skirpichev** [comment]
I think, it's now impossible. "behavior" option has only four options (default, once, new, spammy).  Nothing seems switch off this annoying feature.

BTW, correct link for your config:
https://github.com/beetbox/beets/blob/master/codecov.yml

**Comment by sampsyo** [comment]
Thanks, yes -- I had to rename the file from .yaml to .yml, which may have been the problem? https://github.com/beetbox/beets/pull/1961#issuecomment-214927374

**Comment by sampsyo** [comment]
@codecov, we're having the same problem with pull request status. I followed the docs to disable all PR status messages ([codecov.yml](https://github.com/beetbox/beets/blob/master/codecov.yml)), but we still got all three on this PR: https://github.com/beetbox/beets/pull/1964

Can you please help us disable *all* notifications? Are the docs wrong?

**Comment by untitaker** [comment]
I've just blocked `codecov-io` from the pimutils organization. The yml has no effect at all.

**Comment by waylan** [comment]
>  "behavior" option has only four options (default, once, new, spammy). 

Where is this documented? I couldn't find it anywhere.

**Comment by skirpichev** [comment]
@waylan, [here](https://github.com/codecov/support/wiki/codecov.yml#variable-descriptions) it is.  It's WIP documentation, but I don't know yet anything better about "comments" option.

**Comment by pganssle** [comment]
> @waylan, here it is. It's WIP documentation, but I don't know yet anything better about "comments" option.

If you click on "comments", you'll see that instead of putting a dict under it, the docs say that you can put `off` there to turn off comments entirely. I asked on the chat and the said you can put `False` there. (That said, doesn't seem to be working for me either).

**Comment by stevepeak** [comment]
@skirpichev to disable comment please use this technique:

```yaml
comment: false
```

Thank you!

**Comment by stevepeak** [comment]
Closing as its not an issue ignoring comments. Thank you friends!

**Comment by untitaker** [comment]
This auto-exported config doesn't do what codecov used to do: https://github.com/pimutils/vdirsyncer/blob/master/codecov.yml

On 7 May 2016 15:24:21 CEST, Steve Peak <notifications@github.com> wrote:
>@skirpichev to disable comment please use this technique:
>
>```yaml
>comment: false
>```
>
>Thank you!
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/codecov/support/issues/162#issuecomment-217634710

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by stevepeak** [comment]
@untitaker https://github.com/pimutils/vdirsyncer/pull/445 Cheers!

**Comment by skirpichev** [comment]
@stevepeak, done as you suggested above.

But I still see comments in my pr https://github.com/skirpichev/omg/pull/265.  Also, codecov/changes still active (but they are disabled as suggested in https://github.com/codecov/support/issues/186).

YML looks ok for your parser:
```
$ curl --data-binary @.codecov.yml https://codecov.io/validate
Valid!

{
  "coverage": {
    "precision": 0, 
    "round": "down", 
    "status": {
      "project": {
        "default": {
          "threshold": 1.0
        }
      }, 
      "changes": false, 
      "patch": {
        "default": {
          "target": 90.0
        }
      }
    }
  }, 
  "comment": false
}
```
BTW, pr in question was rebased, if that matters.

**Comment by stevepeak** [comment]
I presume this has been fixed... How are we looking recently?

**Comment by skirpichev** [comment]
Unfortunately, I blocked codecov-io bot from my repos.  Now it's unblocked, I'll check recent PR's.

**Comment by skirpichev** [comment]
No, it's **not fixed**:
https://github.com/diofant/diofant/pull/284#issuecomment-222213502

**Comment by skirpichev** [comment]
@stevepeak, shouldn't this be reopened, or I did something wrong?

**Comment by 153957** [comment]
Also not fixed for us, still getting PR comment: https://github.com/HiSPARC/sapphire/pull/111#issuecomment-222495055
A pull request into which master was merged which contains the .codecov.yml . (.codecov.yml was not in the branch when it was originally created)

**Comment by stevepeak** [comment]
@skirpichev same as above. 

I'll be reviewing this procedure to discover any issues.

**Comment by skirpichev** [comment]
@stevepeak, good news.  But meanwhile I'll ban codecov-io bot again.

**Comment by yoshuawuyts** [comment]
@stevepeak hey, can I disable this without adding a `.codecov.yml` file to my repos just for this purpose? many thanks :sparkles:

**Comment by untitaker** [comment]
I just got comments again, even though I had set `comment: false`. Apparently Codecov's export system exported to `codecov.yml` when it should've been `.codecov.yml`?

**Comment by stevepeak** [comment]
We have a patch in the pipe that we believe will fix this. The issue was a caching issue and the team yaml was ignored in some edge cases. Deploying this week.

**Comment by untitaker** [comment]
This is still an issue for me. Here is my codecov.yml: https://github.com/pimutils/vdirsyncer/blob/master/.codecov.yml

**Comment by untitaker** [comment]
Also the other configuration values I set get completely ignored. The status gets set even though I disabled it.

**Comment by sampsyo** [comment]
Me too, but it seems to be very intermittent. Here's a recent comment: https://github.com/beetbox/beets/pull/2198

But we've had other PRs since then that haven't gotten the prohibited comment.

**Comment by stevepeak** [comment]
Yea we have a couple reports of this going on. It's a very perplexing issue that we are trying to understand. Thought it was aggressive yaml caching but recent commits attempting to fix did not resolve this.

**Comment by stevepeak** [comment]
@sampsyo https://github.com/beetbox/beets/blob/master/codecov.yml#L4 **True**

**Comment by sampsyo** [comment]
For what it's worth, we're still getting comments at seemingly random intervals: https://github.com/beetbox/beets/pull/2298#issuecomment-264699578

**Comment by Dantemss** [comment]
This is just a guess, but maybe the service is looking at the `.codecov.yml` file in the PR commits without performing a merge, which doesn't work if the PR branch is based off of a commit that happened before that file was merged into master.

**Comment by Dantemss** [comment]
Nvm, that doesn't seem to be the issue... it's still ignoring the file.

**Comment by stevepeak** [comment]
This issue was resolved months ago, forgot to close the ticket. Thank you!
